src/App/TSBundle/Entity/Editors.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace App\TSBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
```

Comment:
you use old Doctrine? I have already fix this in last doctrine version - remove this use if you use yml configuration